gipc/gipc.py

```diff
@@ -26,28 +26,33 @@
 import logging
 import multiprocessing
 import multiprocessing.process
-import multiprocessing.reduction
 from itertools import chain
+
 try:
     import cPickle as pickle
 except ImportError:
     import pickle
+
 WINDOWS = sys.platform == "win32"
+
 if WINDOWS:
     import msvcrt
+    import platform
+
+    if platform.python_implementation() == 'PyPy':
+        raise RuntimeError('gevent & gipc are not supported on PyPy on Windows')
```

Comment:
This is something I decided against hard-coding. If a user tries that combination it might even work (when using a subset of functionality).